CONTRIBUTING.md

```diff
@@ -89,10 +89,10 @@ General guidelines for pull requests (PRs):
 To make this test you need to install `pytest` and run:
 
 ```bash
-make test_mandyoc
+make test
 ```
 
-*NOTE*: you might need to manually set the `MPIREXEC` in the Makefile.
+_NOTE_: you might need to manually set the `MPIREXEC` in the Makefile.
```

Comment:
```suggestion
_NOTE_: you might need to manually set the `MPIEXEC` in the Makefile.
```
Currently, we are using mpirun. Does setting MPIEXEC is enough? I think we should use $MPIEXEC instead?